Unable to determine the status of the running process in LogIT without resteasy

Not sure if this is my env specific, but this test fails consistently on my machine
```
2021-06-07 10:51:42,148 DEBUG [org.apa.cam.qua.cor.dep.CamelProcessor] (build-7) Camel XML capability not detected, disable XML reifiers
======================================================================================
  native.image.path was not set, making a guess for the correct path of native image
  guessed path: /Users/ffang/projects/camel-quarkus-new/integration-test-groups/foundation/log/target/camel-quarkus-integration-test-log-2.0.0-SNAPSHOT-runner
======================================================================================
Executing [/Users/ffang/projects/camel-quarkus-new/integration-test-groups/foundation/log/target/camel-quarkus-integration-test-log-2.0.0-SNAPSHOT-runner, -Dquarkus.http.port=55497, -Dquarkus.http.ssl-port=55498, -Dtest.url=http://localhost:55497, -Dquarkus.log.file.path=/Users/ffang/projects/camel-quarkus-new/integration-test-groups/foundation/log/target/quarkus.log, -Dquarkus.log.file.enable=true]
2021-06-07 10:51:43,883 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.http.ssl-port" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2021-06-07 10:51:43,886 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.http.port" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2021-06-07 10:51:43,889 INFO  [org.apa.cam.qua.cor.CamelBootstrapRecorder] (main) Bootstrap runtime: org.apache.camel.quarkus.main.CamelMainRuntime
2021-06-07 10:51:43,892 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Routes startup summary (total:0 started:0)
2021-06-07 10:51:43,892 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 3.10.0 (camel-1) started in 0ms (build:0ms init:0ms start:0ms)
2021-06-07 10:51:43,893 INFO  [foo-topic] (main) Exchange[ExchangePattern: InOnly, BodyType: String, Body: Hello foo!]
2021-06-07 10:51:43,894 INFO  [io.quarkus] (main) camel-quarkus-integration-test-log 2.0.0-SNAPSHOT native (powered by Quarkus 2.0.0.CR2) started in 0.021s. 
2021-06-07 10:51:43,894 INFO  [io.quarkus] (main) Profile prod activated. 
2021-06-07 10:51:43,894 INFO  [io.quarkus] (main) Installed features: [camel-core, camel-log, camel-support-common, cdi]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 13.096 s <<< FAILURE! - in org.apache.camel.quarkus.component.log.it.LogIT
[ERROR] info  Time elapsed: 0.022 s  <<< ERROR!
java.lang.IllegalStateException: Unable to determine the status of the running process. See the above logs for details
	at io.quarkus.test.common.LauncherUtil.waitForCapturedListeningData(LauncherUtil.java:74)
	at io.quarkus.test.common.NativeImageLauncher.start(NativeImageLauncher.java:116)
	at io.quarkus.test.junit.IntegrationTestUtil.startLauncher(IntegrationTestUtil.java:130)
	at io.quarkus.test.junit.NativeTestExtension.doNativeStart(NativeTestExtension.java:150)
	at io.quarkus.test.junit.NativeTestExtension.ensureStarted(NativeTestExtension.java:92)
	at io.quarkus.test.junit.NativeTestExtension.beforeAll(NativeTestExtension.java:65)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$8(ClassBasedTestDescriptor.java:368)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:368)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:192)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:136)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

```

--- a/integration-test-groups/foundation/log/src/main/java/org/apache/camel/quarkus/component/log/it/LogResource.java
+++ b/integration-test-groups/foundation/log/src/main/java/org/apache/camel/quarkus/component/log/it/LogResource.java
@@ -16,21 +16,36 @@
  */
 package org.apache.camel.quarkus.component.log.it;
 
+import java.util.Optional;
+
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
 import javax.inject.Inject;
 
 import org.apache.camel.ProducerTemplate;
 import org.apache.camel.quarkus.main.events.AfterStart;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 @ApplicationScoped
 public class LogResource {
 
+    public static final String LOG_MESSAGE = "Lets's fool io.quarkus.test.common.LauncherUtil.CaptureListeningDataReader: Listening on: http://0.0.0.0:";
     @Inject
     ProducerTemplate producerTemplate;
 
+    @ConfigProperty(name = "quarkus.http.test-port")
+    Optional<Integer> httpTestPort;
+    @ConfigProperty(name = "quarkus.http.port")
+    Optional<Integer> httpPort;
+
+    private int getEffectivePort() {
+        final boolean isNativeMode = "executable".equals(System.getProperty("org.graalvm.nativeimage.kind"));
+        Optional<Integer> portSource = isNativeMode ? httpPort : httpTestPort;
+        return portSource.isPresent() ? portSource.get().intValue() : 0;
+    }
+
     public void info(@Observes AfterStart event) {
-        producerTemplate.sendBody("log:foo-topic", "Hello foo!");
+        producerTemplate.sendBody("log:foo-topic", LOG_MESSAGE + getEffectivePort());
     }
 
 }

--- a/integration-test-groups/foundation/log/src/test/java/org/apache/camel/quarkus/component/log/it/LogTest.java
+++ b/integration-test-groups/foundation/log/src/test/java/org/apache/camel/quarkus/component/log/it/LogTest.java
@@ -33,7 +33,7 @@ public class LogTest {
     public void info() {
         await().atMost(10L, TimeUnit.SECONDS).pollDelay(100, TimeUnit.MILLISECONDS).until(() -> {
             String log = new String(Files.readAllBytes(Paths.get("target/quarkus.log")), StandardCharsets.UTF_8);
-            return log.contains("[foo-topic]") && log.contains("Body: Hello foo!");
+            return log.contains("[foo-topic]") && log.contains(LogResource.LOG_MESSAGE);
         });
 
     }

--- a/pom.xml
+++ b/pom.xml
@@ -44,6 +44,8 @@
         <camel.docs.branch>camel-${camel.major.minor}.x</camel.docs.branch><!-- The stable branch on which our Antora docs depends -->
 
         <cassandra-quarkus.version>1.0.1</cassandra-quarkus.version><!-- https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
+        <debezium.version>${debezium-version}</debezium.version><!-- https://repo1.maven.org/maven2/io/debezium/debezium-bom/ -->
+        <optaplanner.version>${optaplanner-version}</optaplanner.version><!-- https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
         <quarkiverse.freemarker.version>0.3.0.CR1</quarkiverse.freemarker.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/freemarker/quarkus-freemarker-parent/ -->
         <quarkiverse-minio.version>2.0.0.CR3</quarkiverse-minio.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/minio/quarkus-minio-parent/ -->
         <quarkus.version>2.0.0.CR3</quarkus.version><!-- https://repo1.maven.org/maven2/io/quarkus/quarkus-bom/ -->
@@ -66,7 +68,6 @@
         <commons-lang.version>2.6</commons-lang.version><!-- used by hbase, should be pretty stable as commons-lang is not developed actively anymore -->
         <commons-math3.version>3.6.1</commons-math3.version><!-- Mess in the transitive dependencies of Spark and hbase-testing-util -->
         <curator.version>4.3.0</curator.version><!-- Mess in the transitive dependencies of Spark, Zookeeper and other hadoop related components -->
-        <debezium.version>${debezium-version}</debezium.version>
         <derby.version>10.15.2.0</derby.version><!-- Spark -->
         <eddsa.version>${eddsa-version}</eddsa.version>
         <freemarker.version>2.3.31</freemarker.version><!-- @sync io.quarkiverse.freemarker:quarkus-freemarker-parent:${quarkiverse.freemarker.version} prop:freemarker.version -->
@@ -108,7 +109,6 @@
         <okhttp.version>${squareup-okhttp-version}</okhttp.version>
         <okio.version>${squareup-okio-version}</okio.version>
         <opencensus.version>0.28.0</opencensus.version><!-- mess in google-pubsub deps -->
-        <optaplanner.version>${optaplanner-version}</optaplanner.version>
         <perfmark.version>0.23.0</perfmark.version><!-- mess in google-pubsub deps -->
         <protobuf.version>${protobuf-version}</protobuf.version>
         <retrofit.version>2.5.0</retrofit.version>

--- a/pom.xml
+++ b/pom.xml
@@ -45,7 +45,7 @@
 
         <cassandra-quarkus.version>1.0.1</cassandra-quarkus.version><!-- https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
         <quarkiverse.freemarker.version>0.3.0.CR1</quarkiverse.freemarker.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/freemarker/quarkus-freemarker-parent/ -->
-        <quarkiverse-minio.version>0.2.3</quarkiverse-minio.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/minio/quarkus-minio-parent/ -->
+        <quarkiverse-minio.version>2.0.0.CR3</quarkiverse-minio.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/minio/quarkus-minio-parent/ -->
         <quarkus.version>2.0.0.CR3</quarkus.version><!-- https://repo1.maven.org/maven2/io/quarkus/quarkus-bom/ -->
         <quarkus-google-cloud.version>0.7.0</quarkus-google-cloud.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/googlecloudservices/quarkus-google-cloud-services-bom/ -->
         <quarkus-hazelcast-client.version>1.2.0</quarkus-hazelcast-client.version><!-- https://repo1.maven.org/maven2/com/hazelcast/quarkus-hazelcast-client-bom/ -->

